Cocktails9/Cocktails9/View/BottomRow/ProfileView.swift

```diff
@@ -1,18 +1,206 @@
+                    PhotosPicker(
+                        selection: $selectedItem, // Bind the selected item
+                        matching: .images, // Limit picker to images
+                        photoLibrary: .shared()) {
+                            if let imageData = selectedImageData,
+                               let image = UIImage(data: imageData) {
+                                Image(uiImage: image)
+                                    .resizable()
+                                    .scaledToFill()
+                                    .frame(width: 100, height: 100)
+                                    .clipShape(Circle())
+                            } else {
+                                Circle()
+                                    .fill(Color.gray.opacity(0.3))
+                                    .frame(width: 100, height: 100)
+                                    .overlay(
+                                        Text(user.username.prefix(1)) // Display first letter of username if no image
+                                            .font(.largeTitle)
+                                            .bold()
+                                            .foregroundColor(.white)
+                                    )
+                            }
+                        }
+                        .onChange(of: selectedItem) { newItem, _ in
+                            // Handle image selection
+                            Task {
+                                // Retrieve selected asset
+                                if let selectedItem, let data = try? await selectedItem.loadTransferable(type: Data.self) {
+                                    selectedImageData = data
+                                    user.profilePhotoData = data
+                                    UserManager.saveUser(user)
+                                }
+                            }
+                        }
+                        .padding(.trailing, 20)
+                    
+                    VStack(alignment: .leading, spacing: 5) {
+                        Text(user.username)
+                            .font(.title2)
+                            .fontWeight(.semibold)
+                        
+                        Text(user.email)
+                            .font(.subheadline)
+                            .foregroundColor(.gray)
+                    }
+                    Spacer()
+                }
+                .padding(.top, 20)
+                
+                Divider()
+                
+                // Editable Username Section
+                VStack(alignment: .leading, spacing: 12) {
+                    Text("Username")
+                        .font(.title3)
+                        .fontWeight(.bold)
+                        .foregroundColor(.primary)
+                    
+                    if isEditingUsername {
+                        TextField("Enter new username", text: $editedUsername)
```

Comment:
Create new file for text field style and apply it in code like this:
https://thehappyprogrammer.com/custom-textfield-in-swiftui

That will make code more readable and reusable